projects/ui/src/clr-ui-dark.scss

```diff
@@ -4,5 +4,4 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-// Import from clr-angular to preserve file structure until sass theming is removed
-@import '../../angular/src/dark-theme';
+@use '@cds/core/styles/theme.dark.min.css';
```

Comment:
This file used to be its own entry-point that replaces `clr-ui.min.css`. I think we should either delete it (preferable to me) or make it work on its own.

Reply:
Good catch. In that case, I would prefer incorporating the dark theme vars in the single clr-ui.min.css file and tell developers it's built-in (just toggle the cds-theme attribute)

Reply:
Agree.